tutor/src/screens/assignment-edit/ux.js

```diff
@@ -25,14 +26,16 @@ export default class AssignmentUX {
   @observable isShowingConfirmTemplate = false;
   @observable exercises;
   @observable isReady = false;
-  @observable sourcePlanId;
   @observable form;
   @observable activeFilter = 'all';
   @observable templates;
   @observable plan;
   @observable didUserChangeDatesManually = false;
   @observable dueAt;
 
+  // Check if the assignment is old, before WRM release. (No grading template)
+  isCloneOldAssignment;
+
```

Comment:
You'll need to make this `@observable` otherwise mobx won't re-render if it's changed

Reply:
it is only set in the constructor so I thought it does not need the `@observable`.